chapters/hpmor-chapter-013.tex

```diff
@@ -150,15 +150,34 @@ \chapter{Asking the Wrong Questions}
 
 Harry shrugged despairingly and went over to the cabinet he’d been given near his bed, to get his wizard’s robes for the day.
 
-On the floor of the cabinet, under his robes, were the cereal bars, and a note: \begin{align*} \hbox{\scshape Points spent: }&\hbox{1}\\ \hbox{\scshape Current points: }&\hbox{97}\\ \hbox{\scshape You have gained: }&\hbox{\scshape 6 cereal bars}\\ \hbox{\scshape You are still wearing: }&\hbox{\scshape Pyjamas}\\[1.5ex] \multicolumn{2}{c}{\hbox{\scshape Do not eat while you are wearing your pyjamas}}\\ \multicolumn{2}{c}{\scshape You will get a Pyjama Penalty} \end{align*}
+On the floor of the cabinet, under his robes, were the cereal bars, and a note:
+\begin{align*} \hbox{\scshape Points spent: }&\hbox{1}\\
+\hbox{\scshape Current points: }&\hbox{97}\\
+\hbox{\scshape You have gained: }&\hbox{\scshape 6 cereal bars}\\
+\hbox{\scshape You are still wearing: }&\hbox{\scshape Pyjamas}\\%
```

Comment:
I don't understand the `%` at the end of this line, as `[1.5ex]` is an argument to `\\`.

Reply:
fixed